src/ResourceManager/Advisor/Commands.Advisor.Test/ScenarioTests/DisableAzureRMAdvisorRecommendation.ps1

```diff
@@ -0,0 +1,90 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Neagtive Test, no user input for the parameters. 
+#>
+function Disable-AzureRmAdvisorRecommendationNoParameterSet
+{
+		Assert-ThrowsContains { Disable-AzureRmAdvisorRecommendation } "Cannot process command because of one or more missing mandatory parameters: ResourceId."
```

Comment:
This should not throw, it should prompt the user for the correct parameter from the defaultparameterset.

Reply:
No it does not throws, it asks for defaulrParameterSet input from user.